### PR TITLE
Fixing bug

### PR DESCRIPTION
**Pull request by djshah19**

### CODE REVIEW

**Review by apluchik — DISMISSED** [review]

File: services/compute/virtualmachine/wssd_test.go [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+func Test_VirtualMachineValidations(t *testing.T) {
+	wssdcloudclient := client{}
+	httpProxy := ""
+	httpsProxy := ""
+
+	vm := &compute.VirtualMachine{
+		VirtualMachineProperties: &compute.VirtualMachineProperties{
+			OsProfile: &compute.OSProfile{
+				ProxyConfiguration: &compute.ProxyConfiguration{
+					HttpProxy:  &httpProxy,
```
Comment (raghavendra-nataraj) [inline services/compute/virtualmachine/wssd_test.go:19]: Where are we setting the nil value to get the error

File: services/compute/virtualmachine/wssd_test.go [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+					HttpsProxy: &httpsProxy,
+				},
+			},
+		},
+	}
+	err = wssdcloudclient.virtualMachineValidations(wssdcloudproto.Operation_POST, vm)
+	if err == nil {
```
Comment (raghavendra-nataraj) [inline services/compute/virtualmachine/wssd_test.go:44]: We should validate what the error is to make sure  it is failing for that error

**Review by apluchik — APPROVED** [review]

**Review by raghavendra-nataraj — DISMISSED** [review]

File: services/compute/virtualmachine/wssd_test.go [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+	err = wssdcloudclient.virtualMachineValidations(wssdcloudproto.Operation_POST, vm)
+	expectedErrorMsg := "Invalid proxy URL. The URL scheme should be http or https: Invalid Input"
+	assert.EqualErrorf(t, err, expectedErrorMsg, "Error should be: %v, got: %v", expectedErrorMsg, err)
+
+	//nil value
+	vm = &compute.VirtualMachine{
+		VirtualMachineProperties: &compute.VirtualMachineProperties{
+			OsProfile: &compute.OSProfile{
+				ProxyConfiguration: &compute.ProxyConfiguration{
+					HttpProxy:  nil,
```
Comment (raghavendra-nataraj) [inline services/compute/virtualmachine/wssd_test.go:55]: Can we add two test cases, One with http proxy null and other with https

**Review by raghavendra-nataraj — APPROVED** [review]
